### PR TITLE
Supplied elements (2) + Check drafts 

### PR DESCRIPTION
**Pull request by valentinamignosa**

### CODE REVIEW

File: inscriptions/ISic1497.xml [hunk 1]
```diff
@@ -181,7 +181,7 @@
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#ME">
                 <ab>
                     <lb n="1"/>
-                    <supplied reason="undefined" evidence="previouseditor"><expan><abbr>C</abbr><ex>aio</ex></expan> <g type="interpunct">·</g></supplied> Terentio <g type="interpunct">·</g> Symphoro
+                    <supplied reason="undefined" evidence="previouseditor"><expan><abbr>C</abbr><ex>aio</ex></expan></supplied> <supplied reason="undefined" evidence="previouseditor"><g type="interpunct">·</g></supplied> Terentio <g type="interpunct">·</g> Symphoro
```
Comment (simonastoyanova) [inline inscriptions/ISic1497.xml:184]: Are we sure the interpunct is supplied by the editor?

File: inscriptions/ISic2832.xml [hunk 2]
```diff
@@ -187,9 +187,9 @@
...
+                    <lb n="3"/><gap reason="lost" unit="character" extent="unknown"/><supplied reason="lost">ἐγὼ</supplied><unclear>ἐ</unclear>πικαλοῦμ<supplied reason="lost">αι</supplied> <supplied reason="lost">τὸν</supplied> <supplied reason="lost">λαὸν</supplied><gap reason="lost" unit="character" extent="unknown"/>
+                    <lb n="4"/><gap reason="lost" unit="character" extent="unknown"/><supplied reason="lost">ἐκ</supplied> <supplied reason="lost">τ</supplied><unclear>η</unclear>͂ς ἐμῆς π<supplied reason="lost">ροικὸς</supplied> <supplied reason="lost">βοηθεῖν</supplied><gap reason="lost" unit="character" extent="unknown"/>
+                    <lb n="5"/><gap reason="lost" unit="character" extent="unknown"/><supplied reason="lost">τοῖς</supplied><unclear>π</unclear>ένησι · κα<supplied reason="lost">ὶ</supplied> <supplied reason="lost">ὀρφανοῖς</supplied> (?) <supplied reason="lost">τῶν</supplied><gap reason="lost" unit="character" extent="unknown"/>
```
Comment (simonastoyanova) [inline inscriptions/ISic2832.xml:192]: Put a space after τοῖς and `@cert="low"` on the supplied with the question mark.

File: inscriptions/ISic1497.xml (hunk 1 above)
Comment (valentinamignosa) [reply inscriptions/ISic1497.xml:184]: I have to check. That's what indicated in the text before my changes...

File: inscriptions/ISic2832.xml (hunk 2 above)
Comment (valentinamignosa) [reply inscriptions/ISic2832.xml]: Done. But...do you see something odd in the formatting on the second line?

Comment (simonastoyanova) [reply inscriptions/ISic2832.xml]: Uh oh, I did it too, let me see what it looks like now :-)

**Pull request merged**